src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -139,11 +139,20 @@ public int MaxModelValidationErrors
 
         /// <summary>
         /// Gets or sets the flag which causes content negotiation to ignore Accept header 
-        /// when it contains the media type */*. <see langword="false"/> by default.
+        /// when it contains the media type */*. <c>false</c>c> by default.
         /// </summary>
         public bool RespectBrowserAcceptHeader { get; set; }
 
         /// <summary>
+        /// Gets or sets a flag to enable content-negotiation to override formatter match based on
+        /// a request's Accept and Content-Type headers. If set to <c>true</c>, content-negotiation will
```

Comment:
Same comment. Need to explain *what* it means to override and in what cases it applies